ui_src/src/domain/functions/components/functionBox/index.js

```diff
@@ -57,71 +63,77 @@ function FunctionBox({ funcDetails }) {
+                                    {funcDetails.owner === OWNER && <MemphisFunctionIcon alt="Memphis function icon" height="15px" />}
+                                    <owner is="x3d">{functionDetails?.owner === OWNER ? 'Memphis.dev' : functionDetails?.owner}</owner>
+                                </div>
+                                <Divider type="vertical" />
+                                <downloads is="x3d">
+                                    <BiDownload className="download-icon" />
+                                    <label>{Number(1940).toLocaleString()}</label>
+                                </downloads>
+                                <Divider type="vertical" />
+                                <rate is="x3d">
+                                    <Rate disabled defaultValue={2} className="stars-rate" />
+                                    <label>(93)</label>
+                                </rate>
+                                <Divider type="vertical" />
+                                <commits is="x3d">
+                                    <FiGitCommit />
+                                    <label>Commits on {parsingDate(functionDetails?.last_commit, false, false)}</label>
+                                </commits>
+                            </deatils>
                         </div>
-                        {funcDetails.owner === OWNER && <MemphisFunctionIcon alt="Memphis function icon" />}
-                    </div>
 
-                    <div className="function-details">
-                        <div className="function-repo">
-                            <GithubBranchIcon alt="github-branch-icon" />
-                            <OverflowTip text={`${functionDetails?.repository} - ${functionDetails?.branch}`} maxWidth={'150px'}>
-                                {functionDetails?.repository} - {functionDetails?.branch}
-                            </OverflowTip>
-                        </div>
-                        <div className="function-code-type">
-                            <CodeBlackIcon alt="code-icon" />
-                            <OverflowTip text={`${functionDetails?.language}`} maxWidth={'150px'}>
-                                <span>{functionDetails?.language}</span>
-                            </OverflowTip>
+                        <div>
+                            <Button
+                                width="100px"
+                                height="34px"
+                                placeholder={'Install'}
+                                colorType="white"
+                                radiusType="circle"
+                                backgroundColorType="purple"
+                                fontSize="12px"
+                                fontFamily="InterSemiBold"
+                                disabled={(isCloud() && !integrated) || !isCloud()}
+                                // isLoading={loader}
```

Comment:
do we need it?

Reply:
I set it to false, shouldn't we put a loader on this async task?